### ISSUE TITLE
Synkinesis

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#9900]
**Preferred term label:**
Synkinesis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Abnormal involuntary facial movement that occurs with voluntary movement of a different facial muscle group. PMID: 18470836.

**Parent term (use hpo.jax.org/app)**
Abnormality of facial musculature - HP:0000301
Abnormality of eye movement - HP:0000496

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Comment**
Patients experience hypertonic contractures and synkinetic movements such as eye closure with volitional movement of the mouth or midfacial movement during volitional or reflexive eye closure. PMID: 18470836.

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

**Comment by pnrobinson** [comment]
@jagcsun added, thanks
